spec/akamai_cloudlet_manager/base_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+
+RSpec.describe AkamaiCloudletManager::Base do
+
+  describe '#initialize' do
+    subject {
+      AkamaiCloudletManager::Base.new({
+        path_to_edgerc: AkamaiCloudletManager.spec + '/test_edgerc',
+        section:        'test_edgerc'
+      })
+    }
+
+    describe 'attributes' do
+      it 'http_host' do
+        p subject.instance_variable_get(:@http_host)
+        expect(subject.instance_variable_get(:@http_host).class).to eq(Akamai::Edgegrid::HTTP)
```

Comment:
for readability maybe we can use [be_kind_of](https://relishapp.com/rspec/rspec-expectations/v/3-7/docs/built-in-matchers/type-matchers) api.

```ruby
expect(subject.instance_variable_get(:@http_host)).to be_kind_of(Akamai::Edgegrid::HTTP)
```